draft-ietf-quic-recovery.md

```diff
@@ -1282,15 +1315,28 @@ Invoked by loss detection from DetectLostPackets when new packets
 are detected lost.
 
 ~~~
+   InPersistentCongestion(newest_lost_packet):
+     pto = smoothed_rtt + 4 * rttvar + max_ack_delay
+     congestion_period =
+       pto * (2 ^ kPersistentCongestionThreshold - 1)
+     // Determine if all packets in the window before the
+     // newest lost packet, including the edges, are marked
+     // lost
+     return IsWindowLost(newest_lost_packet, congestion_period)
+
    OnPacketsLost(lost_packets):
      // Remove lost packets from bytes_in_flight.
      for (lost_packet : lost_packets):
        bytes_in_flight -= lost_packet.size
-     largest_lost_packet = lost_packets.last()
+     newest_lost_packet = lost_packets.last()
```

Comment:
Elsewhere we use largest(ie: largest_acked) and there are currently no uses of newest in the recovery draft, so I'm going to change this back to largest.  If we want to start using newest, let's do that in a separate PR.

Reply:
Fine with me. I might look at changing it later.